charts/modify-vm-resources/INSTALL.md

```diff
@@ -73,7 +73,33 @@ You can patch the resource by running (update it if needed with your own values)
 
 #### Environment variables
 
-##### ConfigMap
+
+#### Secret
+
+We also need to set the following environment variables:
+* JIRA_API_TOKEN
+* OCP_API_SERVER_TOKEN
+
+To do so, edit the secret `${WORKFLOW_NAME}-creds` and set those values and the one of `NOTIFICATIONS_BEARER_TOKEN`:
+```
+WORKFLOW_NAME=modify-vm-resources
+oc -n sonataflow-infra patch secret "${WORKFLOW_NAME}-creds" --type merge -p '{
+   "data":{
+      "NOTIFICATIONS_BEARER_TOKEN":"'$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}')'"
+   },
+   "stringData":{
+      "JIRA_API_TOKEN":"{{ JIRA_API_TOKEN }}",
+      "OCP_API_SERVER_TOKEN":"{{ OCP_API_SERVER_TOKEN }}"
+   }
+}'
+```
+If you are using Jira cloud, you can generate the `JIRA_API_TOKEN` using https://id.atlassian.com/manage-profile/security/api-tokens 
+
+The `OCP_API_SERVER_TOKEN` should be associated with a service account.
+
+Once the secret is updated, to have it applied, the pod shall be restarted. Note that if you are following the next section, no need for manual restart as any change to the sonataflow CR will restart the pod.
```

Comment:
s/shall/should

Reply:
> 'Shall' is commonly used in official documents and formal contexts to indicate obligations and requirements, particularly when discussing legal or mandatory actions. 'Should', on the other hand, is used to offer advice or recommendations or suggest a course of action, particularly in situations where there is some degree of flexibility or discretion. The choice between 'shall' and 'should' depends on the level of obligation being conveyed

https://langeek.co/en/grammar/course/692/shall-vs-should

So here I want to express an obligation, hence the use of `shall` instead of `should`

Reply:
I see what you mean. The confusing part here is that this wording gives the impression that the pod will be restarted(shall be) automatically, which is not the case.

Reply:
I see, I'll add a word to explicitly say that the restart is not automated when updating the secret

Reply:
Is that better now?